components/BoardMember.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <li>
+        <div class="md:grid md:grid-cols-3 lg:gap-8 sm:pb-2">
+            <div>
+                <img
+                    class="object-cover shadow-lg rounded-lg md:h-60"
+                    :src="`${imageLink}`"
+                    alt=""
+                />
+            </div>
+
+            <div class="md:col-span-2">
+                <div class="space-y-1">
+                    <div class="text-lg font-medium">
+                        <h3>{{ name }}</h3>
+                        <p class="text-indigo-600">{{ title }}</p>
+                        <div v-if="twitterHandle" class="flex pb-1">
+                            <span class="sr-only">Twitter</span>
+                            <a :href="'http://twitter.com/' + twitterHandle">
```

Comment:
I like how you're using the string interpolation here! :)

Reply:
Do you mind leaving a comment indicating that we're using Vue bind to construct the Twitter path here? It might not be obvious what we're doing to doing to people who aren't familiar to Vue if we're using this shorthand syntax.

Reply:
Done :)